ui/src/containers/AlertPage.js

```diff
@@ -453,7 +443,11 @@ export default function AlertPage() {
       {
         Header: 'Name',
         accessor: 'labels.alertname',
-        cellStyle: { width: '300px' },
+        cellStyle: {
+          width: '300px',
+          overflow: 'hidden',
+          textOverflow: 'ellipsis',
+        },
```

Comment:
I think we should still try to use the new table because this fix is not fully solving the problem. The user won't be able to see the full alert name, I think the right implementation should be, if it is ellipsis then hovering it should display a tooltip with the complete alertName. IIRC this is the default behaviour we implemented in the new table component.

Reply:
For what i can see in core-UI storybook it's not the default behavior for the table, we will have to code it anyway. But i can definitly migrate to the new table. I just think it's 2 differents topics.